src/directives/mmsCkEditor.js

```diff
@@ -569,6 +570,33 @@ function mmsCkeditor($uibModal, $templateCache, $timeout, growl, CKEDITOR, _, Ca
         function _isShiftKeyOn(keyboardEvent) {
             return keyboardEvent.shiftKey;
         }
+
+        function _addContextMenuItems(editor) {
+            _addFormatAsCodeMenuItem(editor);
+        }
+
+        function _addFormatAsCodeMenuItem(editor) {
+            editor.addCommand('formatAsCode', {
+                exec: function (editor) {
+                    var selected_text = editor.getSelection().getSelectedText();
+                    var newElement = new CKEDITOR.dom.element("code");
+                    newElement.setStyles({'background-color': '#e8d4e8', color: 'black'});
```

Comment:
shouldn't be setting styles inline, use the ckeditor content css, i think browsers already have a default style for code

Reply:
the browser does have a default style for code, but all it does is changing the text font-family to monospace, so I just added more styling based on the request on the ticket and to just show that it can be done through that api. let me know if we only want default browser styling. Also i moved the inline style to content.css